2017/kickstart.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: news
+title: 2017 Kickstart
+permalink: /2017/kickstart
+date: 2016-10-01
+---
+
+Kickstart is the event which kicks off a competition year, where the game will be announced, and robotics kits will be handed out.
+
+The venues for Student Robotics 2017 kickstart are:
+
+- University of Southampton
+- University of Bristol
+- The Kings Centre, Oxford
+
+Teams are welcome to attend any venue they wish. However; they should inform their local team co-ordinators which venue they will attend, as we will need to arrange for the robotics kits to be at the venue they choose, and some venues have vehicle access restrictions which will need to be arranged beforehand.
+
+Below is an incomplete table of venue-dependent information, which will be completed very soon.
+
+| Venue                     | How to get there                                     | Requires a laptop? | Nearby Food Locations | Finish Time |
+|---------------------------|------------------------------------------------------|--------------------|-----------------------|-------------|
+| University of Southampton | [Getting to campus][soton-campus-directions]. on-campus directions **coming soon** | No                 | [Map][soton-food-map] | 17:00 |
+| University of Bristol     | [Getting to campus][bris-campus-directions]. on-campus directions **coming soon** | Yes                | Coming soon  | Coming soon |
+| The Kings Centre, Oxford  | [Download directions][oxford-directions]             | Yes                | Coming soon | Coming soon |
+
+If teams cannot attend any venues, they should arrange for kit to be couriered to them. For some venues, teams will need to bring laptops or similar computing devices. internet access will be available. Teams will also be expected to make their own provisions for lunch.
+
+Here is a schedule for the day:
+
+|  Time  | Info |
+|--------|------|
+| 10:00  | Introductory presentation, where the game of 2017 will be announced. |
+| 11:00  |Team kits will be handed out, and teams will take part in activities to get them acquainted with their kits. |
+| 12:30  | There will be a lunch break. Teams are expected to bring their own lunches or get them from food vendors near the venues. |
+|16:00 / 17:00| The activities will finish |
+
+[soton-food-map]: https://goo.gl/Np9L1e
+
+[oxford-directions]: {{ '/resources/2017/Oxford_kickstart_directions.pdf' | prepend: site.baseurl }}
+
+[soton-campus-directions]: http://www.southampton.ac.uk/about/visit/getting-to-our-campuses.page#highfield
+[bris-campus-directions]: http://www.bris.ac.uk/maps/directions/
```

Comment:
Inconsistent empty newlines (though I line the idea of a blank line between groups of related links).